app/src/main/res/layout/activity_sign_up.xml

```diff
@@ -0,0 +1,242 @@
+
+    <TextView
+        android:id="@+id/fill_profile"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:layout_marginTop="@dimen/fill_profile_margin"
+        android:text="@string/text_fill_profile"
+        android:textColor="@color/white"
+        android:textSize="@dimen/text_size_fill_profile"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toBottomOf="@+id/greetings" />
+
+    <TextView
+        android:id="@+id/text_view_email"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:layout_marginStart="@dimen/margin_text_view_email"
+        android:layout_marginTop="@dimen/margin_text_view_email"
+        android:labelFor="@+id/edit_text_email"
+        android:text="@string/e_mail"
+        android:textColor="@color/white"
+        android:textSize="@dimen/text_size_email"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toBottomOf="@+id/fill_profile" />
+
+    <EditText
+        android:id="@+id/edit_text_email"
+        android:layout_width="match_parent"
+        android:layout_height="@dimen/height_edit_text_email"
+        android:layout_marginStart="@dimen/margin_edit_text_email"
+        android:layout_marginEnd="@dimen/margin_edit_text_email"
+        android:fadeScrollbars="true"
+        android:inputType="textEmailAddress"
+        android:maxEms="20"
+        android:text=""
+        android:textColor="@color/white"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toBottomOf="@+id/text_view_email" />
+
+    <TextView
+        android:id="@+id/text_valid_email"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:layout_marginStart="@dimen/margin_text_valid_email"
+        android:layout_marginEnd="@dimen/margin_text_valid_email"
+        android:textColor="@color/red"
+        android:text=""
+        android:textSize="@dimen/hint_text_size_email"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toBottomOf="@+id/edit_text_email" />
+
+    <TextView
+        android:id="@+id/text_view_password"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:layout_marginStart="@dimen/margin_text_view_password"
+        android:layout_marginTop="@dimen/margin_top_text_view_password"
+        android:labelFor="@+id/edit_text_password"
+        android:text="@string/password"
+        android:textColor="@color/white"
+        android:textSize="@dimen/text_size_text_view_password"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toBottomOf="@+id/text_valid_email" />
+
```

Comment:
https://github.com/material-components/material-components-android/blob/master/docs/components/TextField.md

need to use it

Reply:
Why do you think it's not by design? I just used different components.

Reply:
Because the fields have different states and always must be a single color. In the design, it is referred to as 'Gray text 2

Reply:
Fixed it